app.yaml

```diff
@@ -0,0 +1,5 @@
+runtime: nodejs
```

Comment:
You're right. It looks like GAE does support environment variables: https://cloud.google.com/appengine/docs/flexible/nodejs/configuring-your-app-with-app-yaml#Node.js_app_yaml_Defining_environment_variables. I'll update.